src/SleepForDays/SleepForDays.workflow.cs

```diff
@@ -0,0 +1,33 @@
+namespace TemporalioSamples.SleepForDays;
+
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+[Workflow]
+public class SleepForDaysWorkflow
+{
+    private bool isComplete;
+
+    [WorkflowRun]
+    public async Task RunAsync()
+    {
+        isComplete = false;
+        while (!isComplete)
+        {
+            await Workflow.ExecuteActivityAsync(
+                (SleepForDaysActivities act) => act.SendEmail("Sleeping for 30 days"),
```

Comment:
Can be a bit confusing to users for the send-email subject to be what the workflow is doing next, but not a big deal